src/mixer.c

```diff
@@ -545,15 +545,18 @@ void mixTable(void)
     }
 
     // motors for non-servo mixes
-    if (numberMotor > 1)
-        for (i = 0; i < numberMotor; i++)
+    if (numberMotor > 1) {
+        for (i = 0; i < numberMotor; i++) {
             motor[i] = rcCommand[THROTTLE] * currentMixer[i].throttle + axisPID[PITCH] * currentMixer[i].pitch + axisPID[ROLL] * currentMixer[i].roll + -cfg.yaw_direction * axisPID[YAW] * currentMixer[i].yaw;
-
-    if (f.FIXED_WING) {
-        if (!f.ARMED)
-            motor[0] = mcfg.mincommand; // Kill throttle when disarmed
-        else
-            motor[0] = constrain(rcCommand[THROTTLE], mcfg.minthrottle, mcfg.maxthrottle);
+            if (f.FIXED_WING) { // vector_trust handeling
+                if (cfg.fw_vector_trust) {
+                    if (f.PASSTHRU_MODE)
+                        motor[i] = rcCommand[THROTTLE] - rcCommand[YAW] * (i - 0.5f);
+                } else { // Override mixerTrustVector
+                    motor[i] = rcCommand[THROTTLE];
+                }
+            }
+        }
     }

```

Comment:
I guess it's vector THRUST.
The function is.
For Fixed Wing vector THRUST is normally active.
Yaw mixed to motors for "Rudder" function.
In Passthru The axisPids is overwritten by Yaw from RC only.

if  cfg.fw_vector_trust = false
Pure Throttle command will be forced to motor[i]

The for loop is used to write motor[i] =